server/controller/propertyController.js

```diff
@@ -54,6 +54,52 @@ class property {
       }
     });
   }
+  static updateProperty(req, res) {
+    const id = parseInt(req.params.id, 10);
+    const theproperty = Property.find(oneProperty => oneProperty.id === id);
+    if (!theproperty) {
+      res.json({
+        status: 'failed',
+        error: 'Could not Update Property',
+      });
+    } else {
+      if (req.files.photo) {
+        const file = req.files.photo;
+        cloudinary.uploader.upload(file.tempFilePath, (error, result) => {
+          if (error) {
+            res.json({
+              status: 'failed',
+              error: 'Could not upload image',
+            });
+          } else {
+            theproperty.image_url = result.url;
+          }
+        });
+      }
+      const {
+        type, state, city, address, price,
+      } = req.body;
+      theproperty.type = type;
+      theproperty.state = state;
+      theproperty.city = city;
+      theproperty.address = address;
+      theproperty.price = price;
+
+      res.json({
+        status: 'true',
```

Comment:
`status` is supposed to be the success status code (Eg. 200, 201, etc) not `true` or `false`.